# Task-1:-

- Matt wants to wake up early in NYC to get used to the local time in france in prior to his travel.
help matt to develop a python code that shows current time in france when he enters the local time in NYC as 22hrs.
- note: NYC is 6 hrs behind of france.

In [2]:
NYC=22

In [3]:
France=(NYC+6)

In [76]:
Time= print(France-24,'am')

4 am


Interpretation :-
   - The current time in the France is 4am at morning.
   - The time differace between NYC and France is 6 hrs.

- ==============================================================================================================================

# Task-2:-

- Matt wants to carry 500 euro wich he wants to get it from the locl currency exchange agent in NYC. 
- There is 1.75% service charge to currency exchange.
- How many doller matt need to spend 500 euros including 1.75% service charge.
- Note: Assume 1.1 Doller= 1 Euro.

Euro = 1.1'Doller'

In [2]:
service_charge= (1.75/100)*500

In [3]:
print(service_charge,'Euro')

8.75 Euro


In [28]:
print(500-8.75,'Euro')

491.25 Euro


In [ ]:
Exchenge rate = 1.1 Doller/Euro

In [27]:
print(491.25*1.1,'dollers')

540.375 dollers


Interpretation :-
- Matt would need to spend 540.38 Dollars to get 500 Euros. 
- after the 1.75% service charge at the currency exchange which is 8.75$.

- ==============================================================================================================================

# Task-3:-

- Matt figure out that he will spending approx. 20% more money in france in a month that he spends in NYC.
- If he spends $4000 per month (30 month) in NYC how many doller will matt need for his 45 day trip in France.

- 1) France is 20% more than NYC. 

- 2) month(30days)=$4000

- 3) for 45 days=_+20%

In [33]:
daily_expence = 4000/30
print(daily_expence,'Dollers per day')

133.33333333333334 Dollers per day


In [34]:
france_trip = daily_expence*45
print(france_trip,'Dollers')

6000.0 Dollers


In [40]:
percentage= (france_trip/100)*20
print(percentage,'Dollers')

1200.0 Dollers


In [39]:
final_expence=france_trip+percentage
print(final_expence,'Dollers')

7200.0 Dollers


Interpretation :-
    - The total expence for 45 days is around 6000$.
    - The expence differance is 20% between NYC and France which is around 1200$.
    - The Final expence is 7200$ for 45 days.

- ==============================================================================================================================

# Task-4:-

- Matt wants to invest in stocks help him to earn some extra money for his trip to france.
- Matt invested in certain stock that will yeild 5% intrest over a month as mentioned by his agent.
- If matt invested $20000, what will the profit after 1 month.

1) Invested = $20,000/-

2) Intrest = 5%

In [46]:
Intrest= (5/100)*20000
print(Intrest,'$')

1000.0 $


In [47]:
Profit = 20000+Intrest
print(Profit,'$')

21000.0 $


Interpretation:-
    - Here we got 1000$ where Intrest rate is 5%.
    - So the total amount about 21000.

- ==============================================================================================================================

# Task-5:-

- Matt wants to practice walking long distance to help him explore tourist places in france.
- He figure out that, on average, an adventure seeking tourist in france walks around 12 Km/day.
- Calculate the number of miles that matt should practice walking per day while he is in NYC.
- Note: 1 km =o.602 miles.

1) walk = 12km/day

2) 1 km = 0.602miles

In [48]:
total_walk= 0.602*12
print(total_walk,'miles/day')

7.224 miles/day


Interpretation :-
    - The average walk in a day is about 7.224 miles per day.

- ==============================================================================================================================

# Task-6:-

- Tour de France bicycle race involves covering 3,328 km approx. 
- Matt is curious to know how many laps an athlete will have to complete in bicycle training facility
- If one lap in the practice track is 1.2 miles long.

1)total covering = 3328km

2)practice track = 1.2miles.

In [53]:
race_track= 3328*0.602
print(race_track,'miles')

2003.456 miles


In [54]:
facility_track=race_track//1.2
print(facility_track,'Laps')

1669.0 Laps


Interpretation:-
    - Total Distance of the racing is 3328km
    - The Distance convert into the miles is 2003.456 miles.
    - Total laps required for covering the distance is 1669 laps.

- ==============================================================================================================================

# Task- 7

- Matt figures out that an average athlete can complete one lap in 2min.
- If the athlete wants to take a 10 min break after every 5 laps, need a 1-hour lunch break.
- Does not want to ride the bicycle for more than 10 hrs a day.
- How many days will the athlete need to complete the distace of 3328km.

1 lap = 2 min

5 lap = 10 min

rest = 10 min

bicycle ride in a day = 10 hrs.

lunch break = 1hr

total ride in a day = 10-1 = 9Hrs.

complete distance = 3328km = 2003.456 miles.

In [60]:
one_hour = 15
print(one_hour,'laps')

15 laps


In [63]:
one_day=one_hour*9
print(one_day,'laps per day')

135 laps per day


- one lap =1.2 miles

In [64]:
Dist_Day= 135*1.2
print(Dist_Day,'miles')

162.0 miles


In [68]:
Total_Days = 2003.456/Dist_Day
print(Total_Days,'Days')

12.367012345679012 Days


Interpretation :-
    - Athlete completes 15 laps in one hour.
    - Using this, Its complete total 135 laps in a single day.
    - when we convert it into distance we got 162 miles per day.
    - Its takes approx 12 days to complete the total distance.

- ==============================================================================================================================

# Task-8:-

- Matt wants to pick up some fresh grape juice from local stores in France. 
- However, the bottle size are marked in 750ml as opposed to ounces(oz) in the US.
- Help Matt calculate the number of botteles of grape juice he needs to buy for 2 weeks.
- If he wants to drink 10 oz juice every night.
note : 1oz= 30ml.

bottle size = 750ml.

2 weeks = 14 days

every night = 10oz = 30ml.

In [77]:
one_day=10
print(one_day,'oz')

10 oz


In [78]:
Two_week= one_day*14
print(Two_week,'oz')

140 oz


In [72]:
convertToml= Two_week*30
print(convertToml,'ml')

4200 ml


In [75]:
Bottles = convertToml/750
print(Bottles)

5.6


Interpretation :-
    - Matt requires 10oz per night which is 30 ml.
    - for two weeks requires 4.2 ltr.
    - when we calculated with bottle size its show 5.6 bottles.
    - so he requires 6 bottles.

- ==============================================================================================================================